x/escrow/keeper/keeper.go

```diff
@@ -97,11 +130,29 @@ func (k *keeper) AccountDeposit(ctx sdk.Context, id types.AccountID, amount sdk.
 		return err
 	}
 
-	if err := k.bkeeper.SendCoinsFromAccountToModule(ctx, owner, types.ModuleName, sdk.NewCoins(amount)); err != nil {
+	// TODO: for escrow accounts created before v0.13:
+	// * should we migrate old escrow accounts to have depositor as owner?
+	// * OR, should we allow old accounts to add a depositor on the first non-owner deposit?
+	// * OR, should they be kept as is?
```

Comment:
Need an answer on this.

Reply:
The first option makes the most sense to me

Reply:
Done